src/index.ts

```diff
@@ -1,27 +1,9 @@
 /* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
Can remove this line

Reply:
not actually, eslint is still confused about react env and supernova env

<img width="1565" alt="Screenshot 2023-05-11 at 15 14 09" src="https://github.com/qlik-oss/sn-pivot-table/assets/29652890/282c135e-e7b8-446a-8002-268f7a4dadea">